host/xtest/regression_1000.c

```diff
@@ -3141,6 +3141,8 @@ static void xtest_tee_test_1037(ADBG_Case_t *c)
 	Do_ADBG_EndSubCase(c, "TA attestation (memory)");
 
 	free_att_key();
+
+	TEEC_CloseSession(&session);
```

Comment:
Good catch. I wonder if it wouldn't make sense to put this just after the `if (...` above since it's not by anything past that point.

Reply:
> Good catch. I wonder if it wouldn't make sense to put this just after the `if (...` above since it's not by anything past that point.
Hi @jenswi-linaro, do you mean to put TEEC_CloseSession directly on line 3126?

Reply:
> do you mean to put TEEC_CloseSession directly on line 3126?

Yes

Reply:
> > do you mean to put TEEC_CloseSession directly on line 3126?
> 
> Yes

OK